### DISCUSSION TITLE
AGI禁令 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#423] [General]
# AGI禁令 - Miao Yu | 于淼

https://yufree.cn/cn/2023/06/24/agi/

**Comment by earfanfan** [comment]
哈哈哈，你这也不能算是脑子里打架的东西吧。因为仅仅只是“从某种角度看赞成或者支持”的观点本来就不能随意迁移应用到任何事物、任何场景上。而且你不总说我们每个人看待当前时代都会受到时代视角的限制嘛。

关于“自洽”这个词，我觉得它并不等于一种完完全全的统一，对任何事物的观点都保持极端的一致可能源于无知，“无知就是力量”，因为现实世界相当相当地复杂，而我们对某一样事物的了解又总是那么有限。当我今天了解了某种事物的一面，觉得它很好，可是过几年事情的发展刷新了我的认知，我可能又觉得不好了，这是很正常的。

另，看电影里面的暴徒在拿机枪扫射的时候是无差别的，不会只是“突突突”，应该是：突突突突突突……
